Robinhood/Robinhood.py

```diff
@@ -757,3 +757,79 @@ def place_sell_order(
         """
         transaction = Transaction.SELL
         return self.place_order(instrument, quantity, bid_price, transaction)
+    
+    ##############################
+    #GET OPEN ORDER(S)
+    ##############################
+    
+    def get_open_orders(self):
+        """
+        Returns all currently open (cancellable) orders.
+        If not orders are currently open, `None` is returned.
+        TODO: Is there a way to get these from the API endpoint without stepping through order history?
+        """
+
+        open_orders = []
+        orders = self.order_history()
+        for order in orders['results']:
+            if(order['cancel'] is not None): 
+                open_orders.append(order)
+
+        if len(open_orders) > 0:
+            return open_orders
+
+    ##############################
+    #CANCEL ORDER(S)
+    ##############################
+
+    def cancel_order(
+            self,
+            orderID
+    ): 
+        """
+        Cancels specified order and returns the response (results from `orders` command). 
+        If order cannot be cancelled, `None` is returned.
+
+        Args:
+            orderID (str): Order ID that is to be cancelled
+
+        """
+        order = self.session.get(self.endpoints['orders'] + orderID).json()
+
+        if order['cancel'] is not None:
+            try: 
+                req = self.session.post(order['cancel'])
+                req.raise_for_status()
+            except requests.exceptions.HTTPError as err_msg:
+                warnings.warn('Failed to cancel order ID: '+ order['id'] + repr(err_msg))
+                return None
+        
+        else: #no cancel link, cannot cancel
```

Comment:
Maybe short circuit this if statement so you just do a

```python
if order['cancel'] is None:
    warnings.warn('Unable to cancel order ID: '+ order['id'])
    return None
```

so you don't have to nest the request logic inside the if block